site/docs/v1/tech/tutorials/integrate-dotnet-api.md

```diff
@@ -0,0 +1,191 @@
+
+```shell
+dotnet publish -r osx-x64
+```
+
+Then run  the executable:
+
+```shell
+ASPNETCORE_ENVIRONMENT=Development bin/Debug/net7.0/osx-x64/publish/MyAPI
+```
+
+You can use [curl](https://curl.se) to see the output of the API: 
+
+```shell
+curl -v http://localhost:5000/identity
+```
+
+This command should return a `401 Unauthorized` response, indicating that the route is secure. The next step is to set up the FusionAuth Hosted Backend API and log in to get a JWT in a cookie that your API can read.
+
+## Create a Login Redirect Page
+
+To demonstrate how simple it is to use the Hosted Backend API, you are going to create a simple login page that redirects to the Hosted Backend API. The login page will be a plain HTML page with no special code or libraries. This page will manage the redirect to the FusionAuth Hosted Backend API login, which will handle the OAuth flow, and then have a button to call your API with the JWT cookie.
+
+Create a new directory for the login page:
+
+```shell
+mkdir LoginPage && cd LoginPage
+```
+
+Then, create two new files: `index.html` and `profile.html`. Add the following code to the `index.html` file:
+
+```html
+{% remote_include https://raw.githubusercontent.com/ritza-co/fusionauth-dotnet-api-example/main/LoginPage/index.html %}
+```
+
+This page is the main page, which has a link that redirects to the FusionAuth Hosted Backend API login route. Note the format of the URL in the anchor `href`: `GET /app/login/{clientId}?redirect_uri={redirectUri}&state={state}&scope={scope}`.
+
+In this case, `clientId` is `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`, as this is hardcoded in the setup script. For applications added to FusionAuth through the UI, you can get the client Id by browsing to the application in FusionAuth and viewing it.
+
+The `redirect_uri` is the URL the Hosted Backend API will redirect to when the login flow is complete, in this case, the `profile.html` page.
+
+The `state` parameter is any information that you would like returned on successful login.
+
+The `scope` is the OAuth scopes you want to request. In this example, the `openid` scope is requested, which is required to get the `app.idt` cookie containing the JWT with the user's claims. You can request other scopes too, such as `offline_access` to get a refresh token, which can be used to get a new access token when the current one expires. The refresh token is stored in a cookie called `app.rt`.
+
+Now add the following to the `profile.html` file:
+
+```html
+{% remote_include https://raw.githubusercontent.com/ritza-co/fusionauth-dotnet-api-example/main/LoginPage/profile.html %}
```

Comment:
I think the Client Id in the refresh token code should be `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e` instead of `deedc32d-70f1-4628-8350-e53b6021eb5d`, right?

Reply:
yes, fixed in the include file